src/pybind/mgr/snap_schedule/fs/schedule_client.py

```diff
@@ -293,6 +308,16 @@ def create_scheduled_snapshot(self,
                     log.info(f'created scheduled snapshot of {path}')
                     log.debug(f'created scheduled snapshot {snap_name}')
                     sched.update_last(time, db)
+                    conn_mgr.dbinfo.reset_path_faults(path)
+                except cephfs.ObjectNotFound:
+                    # maybe path is missing or wrong
+                    self._log_exception('create_scheduled_snapshot')
+                    log.debug(f'path {path} is probably missing or wrong; '
+                              'remember to strip off the mount point path '
+                              'prefix to provide the correct path')
+                    conn_mgr.dbinfo.inc_path_faults(path)
+                    if conn_mgr.dbinfo.get_path_faults(path) >= MAX_PATH_MISSING_FAULTS:
```

Comment:
I though the grace period handling is going to be done as a separate change.

Reply:
This change shouldn't hurt any other functionality. So I added it in this PR.

Reply:
The grace period is not configurable yet.
That would be part of another PR which would require DB schema changes.

Reply:
Well, it does add a grace period - so, schedules which were getting deactivate earlier will remain active now till the limit is reached. IMO, we need to think more about how the grace period should behave when hourly/daily/monthly snapshots schedules are added.

For now let's default to `MAX_PATH_MISSING_FAULTS` as 1, so the earlier behavior is retained. WDYT?

Reply:
Yeah, we could set the default MAX_PATH_MISSING_FAULTS to 1 to retain existing functionality.
My thinking was that since a path was allowed to be added that could come into existence in the future, why not provision a grace period to wait for it rather than deactivating the schedule on the very first attempt.
The new log message provides a reminder as well.

Reply:
It would be made configurable anyway (later), so we can stick to the default for now. If we introduce this grace period, and later, make it configurable, its kind of undoing the hard-coded grace period anyway.